build.sh

```diff
@@ -130,6 +130,13 @@ build_combined() {
     local scope_suffix="$5"
     local version_suffix="$6"
     local config="$CONFIGURATION"
+    local previous_realm_xcode_version="$REALM_XCODE_VERSION"
+    local previous_realm_swift_version="$REALM_SWIFT_VERSION"
+
+    if [[ "$module_name" == "Realm" ]]; then
```

Comment:
it's a local variable, so you're suggesting passing the module name to the function?